graphs/energy/nodes/buildings/buildings_space_heater_combined_hydrogen.converter.ad

```diff
@@ -20,8 +20,9 @@
 - fever.group = buildings_space_heating
 - fever.type = producer
 - fever.technology_curve_type = tech_day_night
-- hydrogen.profile = self: hydrogen_input_curve
+- hydrogen.group = flat
 - hydrogen.type = consumer
+- hydrogen.subtype = generic
```

Comment:
This does not work. Need to rewrite this to merit-like. Will discuss this with @noracato and @kaskranenburgQ on Monday.